### PR TITLE
portfolio: Small feature + controller tests

### PR DESCRIPTION
**Pull request by jmaslek**
# Description

Added tests to the portfolio controller 

Added a new portfolio function: drawdown

mdd

Fixed a quick bug in prediction/mc where -n was not an int type

As an aside: this portfolio menu/ model needs to be refactored.  It is a nightmare to build off of.

### CODE REVIEW

File: gamestonk_terminal/portfolio/portfolio_controller.py [hunk 1]
```diff
@@ -93,6 +85,7 @@ def print_help(self):
 [info]Graphs:[/info][cmds]
     rmr         graph your returns versus the market's returns
     al          displays the allocation of the portfolio
+    mdd         display portfolio drawdown[/cmds]
```
Comment (DidierRLopes) [inline gamestonk_terminal/portfolio/portfolio_controller.py:88]: Why is the name `mdd`?

**Review by DidierRLopes — APPROVED** [review]
Looks good apart from the name `mdd` being confusing to me

File: gamestonk_terminal/portfolio/portfolio_controller.py (hunk 1 above)
Comment (northern-64bit) [reply gamestonk_terminal/portfolio/portfolio_controller.py:88]: I think it's the first letters of "maximum draw down"

**Review by northern-64bit — APPROVED** [review]
This looks good, but the command hasn't been added to the Hugo website in this pr so don't forget it. I agree that the portfolio menu and model needs to be refactored.

File: gamestonk_terminal/portfolio/portfolio_controller.py (hunk 1 above)
Comment (jmaslek) [reply gamestonk_terminal/portfolio/portfolio_controller.py:88]: > I think it's the first letters of "maximum draw down"

@DidierRLopes This. The common jargon

**Comment by jmaslek** [comment]
Commenting that I bumped down pandas to 1.3.5. Pandas at 1.4 seems to broke a bunch a shit

**Pull request merged**